src/modules/TeamsList/components/Teams/components/TeamItem/TeamItem.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC, useState } from 'react';
+import { TeamItemStyled, TeamItemTableWrapper } from './styled';
+import { User } from 'types';
+import { MembersListToggle } from '../MemberListToggle/MemberListToggle';
+import { LIGHT_TEXT_COLOR } from 'appConstants/colors';
+import { TeamUserTable } from '../TeamUserTable/TeamUserTable';
```

Comment:
name file index.tsx and rename import to import { TeamUserTable } from '../TeamUserTable';

Reply:
done